README.md

```diff
@@ -31,3 +30,5 @@ rev chk
 cr,c,c,c,f cr
 
 check2
+
```

Comment:
Inline comment, in the context of a "approve" code review

Reply:
commenting on approved code review.